packages/backend/tsconfig.json

```diff
@@ -2,8 +2,6 @@
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
     "baseUrl": ".",
-    "declaration": true,
```

Comment:
This was already defined in the declarations tsconfig so I removed this duplicate